### DISCUSSION TITLE
ExpressLRS 3.0 support #140

### DISCUSSION THREAD

**Discussion by psicho100pro** [OpenI6X/opentx#183] [Q&A] [answered]
Hi,
I noticed a merged #140
Does this mean that if I connect the Happymodel ES24TX-Lite, for example, it will be possible to control the VRx RX5808 Backpack via ESP8285?
Thank you for answer.

**Comment by ajjjjjjjj** [comment, marked as answer]
Hi, it means that the script will work with new longer field options keeping all current functionality. But this still changes and there will be another patch set later.
Access to "Other devices" like: RX, VTX or even Crossfire as a side effect is not ready yet, but i'm experimenting on it.

**Comment by ajjjjjjjj** [comment]
Other devices support is merged.

**Comment by psicho100pro** [comment]
I'll try when the next necessary hardware arrives, I was scammed by the seller. :D

**Comment by psicho100pro** [comment]
I can confirm the full functionality of ES24TX Lite + EP2 + EP82
